.circleci/config.yml

```diff
@@ -1,5 +1,12 @@
 version: 2
 
+aliases:
+  - &rustup |
+    curl https://sh.rustup.rs -sSf | sh -s -- -y
+    echo 'export PATH="$HOME/.cargo/bin:$PATH"' >> $BASH_ENV
```

Comment:
Can you change this to `echo 'source $HOME/.cargo/env' >> $BASH_ENV`? I didn't understand what was going on here before.